app/main.py

```diff
@@ -1,25 +1,92 @@
 from abc import ABC
+from typing import Type
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Type, name: str) -> None:
+        self.name = name
+
+    def __get__(self, instance: object, owner: Type) -> int:
+        return instance.__dict__.get(self.name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError(f"The value of {self.name} must be an integer")
+        if value < self.min_amount or value > self.max_amount:
+            raise ValueError(f"{self.name}"
+                             f"must be between {self.min_amount}"
+                             f" and {self.max_amount}")
+        instance.__dict__[self.name] = value
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, weight: int,
+                 height: int)\
```

Comment:
The line continuation character `\` is not necessary here because the line break is within parentheses. You can safely remove it.